File: DSA/DSA Using Python/Bubble_sort.py
def BubbleSort(list):
    n = len(list)
    for i in range(n-1,0,-1):
        for j in range(i):
            if list[j]>list[j+1]:
                temp = list[j]
                list[j]=list[j+1]
                list[j+1]=temp

li = [3,5,8,9,6,2]
print('Original Array:',li)
BubbleSort(li)
print('Sorted Array:',li)

File: DSA/DSA Using Go/Bubble_sort.go
package main

import "fmt"

func BubbleSort(slice []int) {
	n := len(slice)
	for pass := n - 1; pass >= 0; pass-- {
		for i := 0; i < pass; i++ {
			if slice[i] > slice[i+1] {
				temp := slice[i]
				slice[i] = slice[i+1]
				slice[i+1] = temp
			}
		}
	}
}

func main() {
	var slice = []int{3, 5, 8, 9, 6, 2}
	fmt.Println("Slice before sorting:", slice)
	BubbleSort(slice)
	fmt.Println("Slice after sorting:", slice)
}
